frontend/pages/_document.tsx

```diff
@@ -0,0 +1,34 @@
+import Document, {
+  DocumentContext,
+  Head,
+  Html,
+  Main,
+  NextScript,
+} from "next/document";
+
+class MyDocument extends Document {
+  public static async getInitialProps(ctx: DocumentContext) {
+    const initialProps = await Document.getInitialProps(ctx);
+    return { ...initialProps };
+  }
+
+  public render() {
+    return (
+      <Html>
```

Comment:
Ist das der Ort wo man auch gleich das lang attribute setzen müsste, oder macht i18next das von selbst?

Reply:
I change the setup a bit now as I was able to solve a problem I had last time.

It actually takes the browser language by default. I set the default language to German for now. We can change that later.
For non-German Browsers the user gets forwarded to a /en/ subroute which is then in english.
The /de/ subroute is also available but isn't used if the browser uses the default language. Not sure if one would want to change that behavior or if it is even possible.